web/download/web-log/index.php

```diff
@@ -27,10 +27,15 @@
 if ($_GET['type'] == 'error') {
     $type = 'error';
 }
-
-exec(HESTIA_CMD."v-list-web-domain-".$type."log $user ".$v_domain." 5000", $output, $return_var);
-if ($return_var == 0) {
-    foreach ($output as $file) {
-        echo $file . "\n";
-    }
+$cmd = implode(" ",array(
+    escapeshellarg(HESTIA_CMD."v-list-web-domain-".$type."log"),
+    escapeshellarg($user),
```

Comment:
Both $user and $domain are allready escapeshellarg causing 

 ''\''jaap'\''' ''\''domain.com'\''' 
 
 https://github.com/hestiacp/hestiacp/blob/18adc9541067a5a4d734fae54be86dc6da56fca1/web/download/web-log/index.php#L8-L9
 
 And 

https://github.com/hestiacp/hestiacp/blob/18adc9541067a5a4d734fae54be86dc6da56fca1/web/inc/main.php#L121-L129

Reply:
thanks, i'll check it out

Reply:
does this look right? https://github.com/hestiacp/hestiacp/pull/2741/commits/842d114e7756bd5d20184d2a5b2e6422554d9a0a